docs/src/guides/smartweave/warp/write-interactions.md

```diff
@@ -0,0 +1,70 @@
+# Warp WriteInteractions
+
+SmartWeave contracts contain a current state and the ability to modify that state via function calls, the way you call a function on a SmartWeave Contract is by creating a transaction known as a SmartWeave Action. This action contains the function name you would like to call and the input parameters required by the SmartWeave Contract function. `contract.writeInteraction` is a function that creates a SmartWeave Action.
```

Comment:
This seems off with the comma.

`...via function calls, the way you call a function...`

 via function calls. The way you call a function